pyatlan/pkg/utils.py

```diff
@@ -0,0 +1,97 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2023 Atlan Pte. Ltd.
+import logging
+import os
+from pathlib import Path
+
+from jinja2 import Environment, PackageLoader
+from pydantic import BaseModel, PrivateAttr
+
+from pyatlan.client.atlan import AtlanClient
+from pyatlan.pkg.models import CustomPackage, RuntimeConfig
+
+LOGGER = logging.getLogger(__name__)
+
+
+class PackageWriter(BaseModel):
+    path: str
+    pkg: CustomPackage
+    _root_dir: Path = PrivateAttr()
+    _config_maps_dir: Path = PrivateAttr()
+    _templates_dir: Path = PrivateAttr()
+    _env: Environment = PrivateAttr()
+
+    def __init__(self, **data):
+        super().__init__(**data)
+        self._root_dir = Path(self.path) / self.pkg.name
+        self._config_maps_dir = self._root_dir / "configmaps"
+        self._templates_dir = self._root_dir / "templates"
+        self._env = Environment(  # noqa: S701
+            loader=PackageLoader("pyatlan.pkg", "templates")
+        )
+
+    def create_package(self):
+        self._root_dir.mkdir(parents=True, exist_ok=True)
+        with (self._root_dir / "index.js").open("w") as index:
+            index.write(CustomPackage.indexJS())
+        with (self._root_dir / "index.js").open("w") as index:
+            index.write(CustomPackage.indexJS())
```

Comment:
Looks familiar... Don't we have this same class and method defined elsewhere already?

Can we dedup, or is this back to the classic Python inability to handle circular dependencies?

Reply:
This was an oversight from some refactoring. This code was moved. I have deleted this.